src/platform/options/default_configuration.cpp

```diff
@@ -295,6 +295,8 @@ void mo::DefaultConfiguration::parse_arguments(
     {
         desc.add_options()
             ("help,h", "this help text");
+		desc.add_options()
+            ("mir-version", "display MIR platform version");
```

Comment:
```suggestion
            ("version,V", "display Mir version and exit");
```

To be frank, off the top of my head I don't recall whether case is differentiated on. `-v` is usually used for `--verbose`, so this will be better, if we can't do `-V`:

```suggestion
            ("version", "display Mir version and exit");
```